configs/milvus.yaml

```diff
@@ -94,6 +94,8 @@ minio:
   region: ""
   # Cloud whether use virtual host bucket mode
   useVirtualHost: false
+  # timeout for request time in milliseconds
+  requestTimeoutMs: 3000
```

Comment:
How about changing the unit of the timeout duration to seconds?

Reply:
sdk default value is 3000 millseconds, and keep this default is better, sdk will retry for 10 times.  change to seconds seems no big advantage and millseonds is more fine-grained.